src/CoreBundle/EventListener/BackendBreadcrumbListener.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\EventListener;
+
+use ChameleonSystem\CoreBundle\Service\BackendBreadcrumbService;
```

Comment:
This class should also use the interface.

Reply:
fixed